package.json

```diff
@@ -53,4 +53,4 @@
     "tslint": "6.1.3",
     "typescript": "5.1.6"
   }
-}
+}
```

Comment:
this intentional?

Reply:
I think it's my auto prettier getting overzealous